kinbot/reaction_generator.py

```diff
@@ -273,12 +266,10 @@ def generate(self):
                         else:
                             e2, st_pt.energy = self.qc.get_qc_energy(str(st_pt.chemid) + '_well')
                             e2, st_pt.zpe = self.qc.get_qc_zpe(str(st_pt.chemid) + '_well')
-                            st_pt.bond_mx()
                             st_pt.characterize(0)  # not allowed to use the dimer option here
-                            st_pt.calc_chemid()
```

Comment:
Same as above comment, do we no longer need to run bond_mx() and calc_chemid()

Reply:
same